Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocAutoCompleteReferenceValueControlObject.cs

```diff
@@ -332,25 +332,34 @@ private SearchServiceResultItem GetFirstAutoCompleteResult ([NotNull] string fil
   data = Sys.Serialization.JavaScriptSerializer.serialize(data);
 
   var returnValue = null;
-  var request = {{
-    async:false,
-    type:'POST',
-    contentType:'application/json; charset=utf-8',
-    url:options.serviceUrl + '/' + options.{3},
-    data:data,
-    dataType:'json',
-    success:function(result, context, methodName){{ returnValue = {{ {4}:'{6}', {5}:result.d }}; }},
-    error:function(error, context, methodName){{ returnValue = {{ {4}:'{7}', {5}:error }}; }}
+  var request = new XMLHttpRequest();
+  request.open('POST', options.serviceUrl + '/' + options.{3}, false);
```

Comment:
I'm wondering why we're plain XmlHttpRequest here instead `Sys.Net.WebServiceProxy.invoke`

Reply:
I did not know about this one but it also seems to be asynchronous, which we do not want in this case.

Reply:
ah, okay. so that's the reason we chose the low-level variant. The thing is, the ajax xmlrequest was always a bit particular when it comes to calling asp.net web services. Please add a note why we can't use the infrastructure-supplied api (sync vs async).